server/services/database-service.js

```diff
@@ -1,7 +1,6 @@
 const Sequelize = require('sequelize')
 const config = require('../config')
 
-module.exports = (async function () {
-  const dbConfig = config.dbConfig[config.env]
-  return new Sequelize(dbConfig.database, dbConfig.username, dbConfig.password, dbConfig)
-}())
+const dbConfig = config.dbConfig[config.env]
+
+module.exports = new Sequelize(dbConfig.database, dbConfig.username, dbConfig.password, dbConfig)
```

Comment:
There is no need to have this module execute an iife as there is no awaitable functions required in the instantiation of it. To that end, I've simplified it to just return an instance of Sequelize which makes the consumers of it less complicated.

Reply:
As discussed, I've updated [`database-service`](https://github.com/DEFRA/ffc-demo-payment-service/blob/544414ee23e42f7b9a6b38a1245a99eb795a1e37/server/services/database-service.js) to export an instance of `sequelize` (used in `db-helper`) and `models`.